### PR TITLE
Adds currach support for Aesry to Halasa Temple

### PR DESCRIPTION
**Pull request by Nituki**
Boards currach if on aesry or halasa island and rows to target. Also added support to row to either target if already on the currach.

### CODE REVIEW

File: bescort.lic [hunk 1]
```diff
@@ -289,12 +293,44 @@ class Bescort
...
+      row_direction = 'pier'
+    elsif mode == 'halasa'
+      row_direction = 'rock'
+    else
+      echo("Unrecognized argument: #{mode}")
+      exit
+    end  
+
+    if (Room.current.id == 5555 || Room.current.id == 5557 || Room.current.id == 5558 || Room.current.id == 15863)
+      fput "go currach", "You climb aboard the", "You are aboard", "What are you referring"
```
Comment (rpherbig) [inline bescort.lic:318]: How do you feel about using `bput` and `move` instead of `fput` for all of this new code?

File: bescort.lic [hunk 2]
```diff
@@ -289,12 +293,44 @@ class Bescort
...
+      jolas(args.mode)
+    elsif args.currach
+      currach(args.mode)
     end
   end
 
   private
 
+  def currach(mode)
+    if (mode.include?('aesry') && (Room.current.id == 5555 || Room.current.id == 5557 || Room.current.id == 5558) || (mode.include?('halasa') && Room.current.id == 15863))
```
Comment (rpherbig) [inline bescort.lic:305]: This conditional is a bit hard to read, in my opinion. What would you think about refactoring it?

A few ideas:

* two `if` blocks, one for aesry and one for halasa
* Using something like `[5555, 5557, 5558].include?(Room.current.id)`

File: bescort.lic [hunk 3]
```diff
@@ -289,12 +293,44 @@ class Bescort
...
+
+    if (Room.current.id == 5555 || Room.current.id == 5557 || Room.current.id == 5558 || Room.current.id == 15863)
+      fput "go currach", "You climb aboard the", "You are aboard", "What are you referring"
+      fput "get oars"
+      fput "untie currach"
+    end
+
+    loop do
+      if bput("tie currach to #{row_direction}", "isn't close enough!", "You pull at the oars") == "You pull at the oars"
+        fput "stand"
```
Comment (rpherbig) [inline bescort.lic:325]: What do you think about using `fix_standing` here?

File: bescort.lic (hunk 1 above)
Comment (Nituki) [reply bescort.lic]: Thanks submitted a cleaner version based off your comments.

File: bescort.lic (hunk 2 above)
Comment (Nituki) [reply bescort.lic]: Great idea!

File: bescort.lic (hunk 3 above)
Comment (Nituki) [reply bescort.lic]: Didn't even know about fix_standing. Updated on v2

**Pull request merged**